Assets/GothicVR/Scripts/World/SunMover.cs

```diff
@@ -0,0 +1,61 @@
+using GVR.Demo;
+using GVR.Util;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SunMover : MonoBehaviour
+{
+    private DebugSettings.SunMovementPerformance sunPerformanceSetting;
+    public GameObject sun;
+
+    public DateTimeGameEvent secondChangeEvent;
+    public DateTimeGameEvent minuteChangeEvent;
+    public DateTimeGameEvent hourChangeEvent;
+
+    private void OnEnable()
+    {
+
+    }
+    private void OnDisable()
```

Comment:
Thanks for thinking about the dynamic on/off feature.
Why not putting the OnEventRaised += ... method to the OnEnable() function as the disable is also on Disable()?